packages/p2p/src/components/order-details/order-details-wrapper.jsx

```diff
@@ -9,6 +9,14 @@ import OrderDetailsFooter from 'Components/order-details/order-details-footer.js
 const OrderDetailsWrapper = ({ children, onPageReturn, page_title }) => {
     const { order_store, sendbird_store } = useStores();
 
+    const showMobileOrderFooter = () => {
```

Comment:
Please move `showMobileOrderFooter` outside the `OrderDetailsWrapper` components. Otherwise, it gonna mount and unmount each time. Maybe rename it to `MobileOrderFooter`.

https://levelup.gitconnected.com/code-review-avoid-declaring-react-component-inside-parent-component-1768a645f523